MergerLogic/ImageProcessing/TileMerger.cs

```diff
@@ -131,7 +131,11 @@ private void AddTileToImageList(Coord targetCoords, Tile? tile, List<MagickImage
             }
 
             var tileBytes = tile.GetImageBytes();
-            MagickImage? tileImage = new MagickImage(tileBytes);
+            MagickImage? tileImage = new MagickImage(tileBytes)
+            {
+                ColorSpace = ColorSpace.sRGB
```

Comment:
this should be always?
don't we need to check the image type first?

Reply:
Nope, it should always be `sRGB`, that's our target `ColorSpace` (see `MergeTiles` aswell).